### DISCUSSION TITLE
How to put items after app bar title?

### DISCUSSION THREAD

**Discussion by Joebayld** [vuetifyjs/vuetify#17873] [Q&A]
I'm trying to figure out what the best slot is to put a button after the App Bar title. Using the `append` slot puts it all at the other end. Some code attached. 

```html
<template>
  <v-app-bar title="Server">
    <template #append>
      <v-menu>
        <template #activator="{ props }">
          <v-btn icon="mdi-dots-horizontal" v-bind="props"></v-btn>
        </template>

        <v-list>
          <v-list-item v-for="(item, i) in 4" :key="i">
            <v-list-item-title>{{ item }}</v-list-item-title>
          </v-list-item>
        </v-list>
      </v-menu>
    </template>
  </v-app-bar>
</template>
```
<img width="707" alt="Screenshot 2023-07-25 at 5 01 21 PM" src="https://github.com/vuetifyjs/vuetify/assets/9399317/bb58244e-bebc-4657-bbed-86102962cb59">

**Comment by xubeiyan** [comment]
you can put `v-btn` in `v-app-bar-title` components, like this
```javascript
<v-app-bar>
  <v-app-bar-title>
     title
    <v-btn icon="$vuetify"></v-btn>
  </v-app-bar-title>
</v-app-bar>
```

the real example like this: https://play.vuetifyjs.com/#eNqlUMtOwzAQ/BVjIQWkOgaVU5VW5cYfcKh7yMMFg1+y3SBU5d9Z26katUgg1SfP7O7M7G4O+Nnast9zvMBV4MrKOvAV0whVPamtTd8jIE3tRmJKkSCCzEP5JXyC0NkEjURr9JLhWzALYvfN8KqiqXKSBPyb5oTP0TKGf0UnmQH61gkbkOdhn6MLZY0L6IAc36EB7ZxRqIAERWxHCCL5gJR/Q8vYcVe8cCkNejVOdjfFfTTIkiCGZzirEVXb8sMbDTc7RBU2FjzDC5SYyIFLxAy/h2D9gtK20zDWcSl6V2oeqLaKrqGNur0OQnHSGbWel/PyiXbChyldcq9I48yX5w5EGJ5NbPI5/7QaO5PD40O2GDki68ZHjwttCoY9d8Rx3XHH3X9XOhubrnVWulgtug9MD3BwKfSnP7t169OdN8eQ1+2c1KLUNjoOM5yCgnWewNsfuAwJTA==

i'm not certain you can visit this link...

**Reply by Joebayld** [reply]
I've done this - it seems to get weird if you have many items and starts to wrap.. 

Seems like a lot of Material sites have buttons after the title - I wonder if it could be a feature request to add a slot after the title?
